ksql-functional-tests/src/test/resources/query-validation-tests/session-windows.json

```diff
@@ -91,10 +91,10 @@
       ],
       "outputs": [
         {"topic": "S2", "key": 0, "value": "0,0", "timestamp": 0, "window": {"start": 0, "end": 30000, "type": "time"}},
-        {"topic": "S2", "key": 0, "value": null, "timestamp": 10000, "window": {"start": 0, "end": 30000, "type": "time"}},
+        {"topic": "S2", "key": 0, "value": null, "timestamp": 0, "window": {"start": 0, "end": 30000, "type": "time"}},
```

Comment:
Shouldn't this be changed to `30000` what is the window end-timestamp?

Reply:
That's not what streams is outputting though...

Reply:
This is probably because this is "legacy" session keys - i.e old / bad session key functionality. Where it set the window end time to the 30 seconds, not the actual session window end. (Which I've fixed)